scripts/commands/update-translations.js

```diff
@@ -52,11 +52,7 @@ export default async function updateTranslations() {
 			`wp i18n make-pot . ${ POT_FILE_PATH } --ignore-domain`
 		);
 
-		if ( makePotResult.stderr ) {
-			error( `wp i18n make-pot failure: ${ makePotResult.stderr }` );
-			return false;
-		}
-
+		// If execPromise doesn't reject the promise, the command was successful
```

Comment:
Is this accurate? Can you provide more details?

Reply:
Here is what occurrs when the command returns an error
![non-existent source directory](https://github.com/Automattic/wc-calypso-bridge/assets/31164683/b2462804-c8de-45e8-ba81-3e980fa36487)

If you run the command in a directory that contains no files, there is no error - the POT file that is created simply has a header
```msgid ""
msgstr ""
"Project-Id-Version: WooCommerce Calypso Bridge 2.2.3\n"
"Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/wc-calypso-bridge\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"POT-Creation-Date: 2023-07-20T18:54:30+00:00\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"X-Generator: WP-CLI 2.6.0\n"```